aspnet/migration/rc1-to-rc2.rst

```diff
@@ -0,0 +1,352 @@
+Migrating from ASP.NET to ASP.NET Core
+======================================
+
+.. contents:: Sections:
+  :local:
+  :depth: 1  
+
+Overview
+--------
+
+This migration guide covers migrating an ASP.NET 5 RC1 application to ASP.NET Core.
+
+ASP.NET 5 RC1 apps were based on the .NET Execution Environment (DNX) and made use of DNX specific features. ASP.NET Core RC2 is based on .NET Core, so you must first migrate your application to the new .NET Core project model. For information on migrating from DNX to .NET Core CLI see <need link>.
```

Comment:
Is this what you are looking for http://dotnet.github.io/docs/core-concepts/dnx-migration.html?